### PR TITLE
Add swagger visualizer to composer

### PR DESCRIPTION
**Pull request by DimalChandrasiri**

### CODE REVIEW

File: composer/modules/web/src/plugins/swagger-visualizer/examples/src/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (DimalChandrasiri) [reply composer/modules/web/src/plugins/swagger-visualizer/examples/src/index.js]: Fixed with commit ce70189

File: composer/modules/web/src/plugins/swagger-visualizer/examples/src/index.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+import { render } from 'react-dom';
```
[the remark this answers is not in the document]
Comment (DimalChandrasiri) [reply composer/modules/web/src/plugins/swagger-visualizer/examples/src/index.js]: Fixed with commit ce70189

File: composer/modules/web/src/plugins/swagger-visualizer/examples/src/index.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
```
[the remark this answers is not in the document]
Comment (DimalChandrasiri) [reply composer/modules/web/src/plugins/swagger-visualizer/examples/src/index.js]: Fixed with commit ce70189

File: composer/modules/web/src/plugins/swagger-visualizer/src/scripts/parameters/parameters.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (DimalChandrasiri) [reply composer/modules/web/src/plugins/swagger-visualizer/src/scripts/parameters/parameters.js]: Fixed with commit ce70189

File: composer/modules/web/src/plugins/swagger-visualizer/src/scripts/index.js [hunk 5]
```diff
@@ -0,0 +1,232 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (DimalChandrasiri) [reply composer/modules/web/src/plugins/swagger-visualizer/src/scripts/index.js]: Fixed with commit ce70189

File: composer/modules/web/src/plugins/swagger-visualizer/src/scripts/operations/add-operation.js [hunk 6]
```diff
@@ -0,0 +1,93 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (DimalChandrasiri) [reply composer/modules/web/src/plugins/swagger-visualizer/src/scripts/operations/add-operation.js]: Fixed with commit ce70189

File: composer/modules/web/src/plugins/swagger-visualizer/src/scripts/operations/operation.js [hunk 7]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (DimalChandrasiri) [reply composer/modules/web/src/plugins/swagger-visualizer/src/scripts/operations/operation.js]: Fixed with commit ce70189

File: composer/modules/web/src/plugins/swagger-visualizer/src/scripts/operations/operations.js [hunk 8]
```diff
@@ -0,0 +1,101 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (DimalChandrasiri) [reply composer/modules/web/src/plugins/swagger-visualizer/src/scripts/operations/operations.js]: Fixed with commit ce70189

File: composer/modules/web/src/plugins/swagger-visualizer/src/scripts/parameters/parameter.js [hunk 9]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (DimalChandrasiri) [reply composer/modules/web/src/plugins/swagger-visualizer/src/scripts/parameters/parameter.js]: Fixed with commit ce70189

File: composer/modules/web/src/plugins/swagger-visualizer/src/scripts/resources/add-resource.js [hunk 10]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (DimalChandrasiri) [reply composer/modules/web/src/plugins/swagger-visualizer/src/scripts/resources/add-resource.js]: Fixed with commit ce70189

File: composer/modules/web/src/plugins/swagger-visualizer/src/scripts/resources/resources.js [hunk 11]
```diff
@@ -0,0 +1,102 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (DimalChandrasiri) [reply composer/modules/web/src/plugins/swagger-visualizer/src/scripts/resources/resources.js]: Fixed with commit ce70189

File: composer/modules/web/src/plugins/swagger-visualizer/webpack.config.js [hunk 12]
```diff
@@ -0,0 +1,43 @@
+const path = require('path');
```
[the remark this answers is not in the document]
Comment (DimalChandrasiri) [reply composer/modules/web/src/plugins/swagger-visualizer/webpack.config.js]: Fixed with commit ce70189

File: composer/modules/web/src/plugins/swagger-visualizer/.eslintrc.js [hunk 13]
```diff
@@ -0,0 +1,62 @@
+module.exports = {
```
[the remark this answers is not in the document]
Comment (DimalChandrasiri) [reply composer/modules/web/src/plugins/swagger-visualizer/.eslintrc.js]: Added with 76e0225

[the remark this answers is not in the document]
Comment (pahans) [reply composer/modules/web/src/plugins/swagger-visualizer/.eslintrc.js]: can we reuse it without copying it?

[the remark this answers is not in the document]
Comment (DimalChandrasiri) [reply composer/modules/web/src/plugins/swagger-visualizer/.eslintrc.js]: Fixed with 9eb4081

**Pull request merged**